src/main/plugins/org.dita.xhtml/xsl/xslhtml/dita2htmlImpl.xsl

```diff
@@ -1078,7 +1079,7 @@
         <xsl:if test="ancestor::*[contains(@class, ' topic/title ')]/parent::*[contains(@class, ' topic/topic ')]">
           <xsl:choose>
             <!-- Not the first one in a title -->
-            <xsl:when test="generate-id(.) != generate-id(key('tm', .)[1])">skip</xsl:when>
+            <xsl:when test="generate-id(.) != generate-id($root/key('tm', .)[1])">skip</xsl:when>
```

Comment:
IMO passing the context as the third argument of `key()` is a bit more familiar to people.